docs/graphql/manual/schema/computed-fields.rst

```diff
@@ -184,6 +184,76 @@ Adding a computed field to a table
      A computed field can be added to a table using the :ref:`add_computed_field <api_computed_field>`
      metadata API
 
+Accessing Hasura session variables in computed fields
+-----------------------------------------------------
+
+It can be useful to have access to the session variable from the SQL function defining a computed field.  For instance, suppose we are recording which users have liked which article, using a table ``article_likes`` specifying a many-to-many relationship between ``article``s and ``user``s.  Then it may be useful to know if the current user liked the current article, and expose this as a *Boolean* computed field on ``article``.
```

Comment:
```suggestion
It can be useful to have access to the session variable from the SQL function defining a computed field.  For instance, suppose we want to record which users have liked which articles. We can do so using a table ``article_likes`` that specifies a many-to-many relationship between ``article`` and ``user``. In such a case it can be useful to know if the current user has liked a specific article, and this information can be exposed as a *Boolean* computed field on ``article``.
```

Reply:
When we talk about the many-to-many relationship, can we link to the respective docs page: https://hasura.io/docs/1.0/graphql/manual/schema/relationships/database-modelling/many-to-many.html?

Reply:
Also, I made the suggestion to remove the `s`s right after the table names, since it screws up the formatting: 
![image](https://user-images.githubusercontent.com/5722022/79969196-710ca580-8491-11ea-8ac9-18b10f932bf1.png)